lib/rust/ensogl/component/grid-view/src/scrollable.rs

```diff
@@ -0,0 +1,83 @@
+//! Module containing scrollable version of [`GridView`].
+
+use crate::prelude::*;
+
+use crate::Entry;
+
+use enso_frp as frp;
+use ensogl_core::application::command::FrpNetworkProvider;
+use ensogl_core::application::Application;
+use ensogl_core::display;
+use ensogl_core::display::scene::Layer;
+use ensogl_scroll_area::ScrollArea;
+
+
+
+// ================
+// === GridView ===
+// ================
+
+/// A template for [`GridView`] structure, where entry parameters and model are separate generic
+/// arguments, similar to [`crate::GridViewTemplate`] - see its docs for details.
+#[derive(CloneRef, Debug, Deref, Derivative)]
+#[derivative(Clone(bound = ""))]
+pub struct GridViewTemplate<E: 'static, M: frp::node::Data, P: frp::node::Data> {
```

Comment:
As requested earlier, please use better names than "E", "M", and "P" ora add docs.